src/flavors/DQMC/slice_matrices.jl

```diff
@@ -53,7 +53,7 @@ const DQMC_CBTrue = DQMC{M, CheckerboardTrue} where M
 
 function slice_matrix(mc::DQMC_CBTrue, m::Model, slice::Int,
 					power::Float64=1.)
-  M = eye(heltype(mc), m.flv*m.l.sites)
+  M = eye(heltype(mc), m.flv*numSites(m.l))
```

Comment:
Is the `eye` function here even functional? It has been dropped from `Base` in 0.7 and we don't seem to define it anywhere. Could you replace those instances by something based on `LinearAlgebra.I`?